galaxy/api/views/views.py

```diff
@@ -109,6 +117,71 @@ def filter_rating_queryset(qs):
 # -----------------------------------------------------------------------------
 
 
+class MonitorRootView(base_views.APIView):
+    """ Monitor resources """
+    permission_classes = (AllowAny,)
+
+    monitor_response = OrderedDict()
```

Comment:
Why `monitor_response` is a class variable?

Reply:
@cutwater I tried to follow the file, I am not a Python developer. Would you like it not to be a class variable?